app/src/main/res/layout/fragment_notification.xml

```diff
@@ -10,17 +10,16 @@
         android:id="@+id/linearLayout"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:gravity="center_horizontal">
+        android:gravity="center_horizontal"
+        android:paddingStart="8dp"
+        android:paddingEnd="8dp">
 
         <TextView
             android:id="@+id/tvNotificationHeader"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
-            android:layout_marginTop="8dp"
-            android:layout_marginBottom="8dp"
+            android:layout_marginTop="16dp"
```

Comment:
Isn't the margin on top of the text too much now and the margin between the text and the button below too little?

Reply:
> Also, I am thinking it will be better to make scrollbar always visible in this page like Filters.
> 
> ```
> android:fadeScrollbars="false"
> ```
> 
> You can do it by adding the above line to scrollview like in fragment_filters.xml file

It's already there

Reply:
Fixed